### PR TITLE
Concourse ci

### PR DESCRIPTION
**Pull request by thullSL**
Builds the front end and pushes it to the s3 bucket

### CODE REVIEW

File: concourse/build.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+platform: linux
+
+image: docker:///thull/bristechfrontbuild
```
Comment (csscottlogic) [inline concourse/build.yml]: Can we get this into the bristechsrm DockerHub repo? We have build-go and build-fsharp in there, so we should probably call it build-front?

**Comment by dwybourn** [comment]
:shipit:

File: concourse/pipeline.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+jobs:
+- name: build
+  public: true
+  serial: true
+  plan:
+  - get: code
+    trigger: true
+  - task: build
+    file: code/concourse/build.yml
+  - put: frontbucket
```
Comment (csscottlogic) [inline concourse/pipeline.yml]: When you "put" a Docker image, you have to give it a ```build``` parameter telling it the directory with the Dockerfile and the build context. How does this know which directory to push to the S3 bucket?

**Comment by csscottlogic** [comment]
Apart from wondering how the "gulped" output becomes the input of bucketChucker, :shipit:

File: concourse/pipeline.yml (hunk 2 above)
Comment (thullSL) [inline concourse/pipeline.yml]: I'm doing that wrong. I though it was getting the output from the previous task. which would be (./gulped). Turns out it gets passed the whole source directory (./).

It's currently pushing EVERYTHING to s3 :open_hands:

**Pull request merged**